### DISCUSSION TITLE
Wrong readings. What am I doing wrong?

### DISCUSSION THREAD

**Discussion by jalla2000** [jomjol/AI-on-the-edge-device#3016] [General]
### Issue

I think I have configured the device well, but I am getting wrong readings...

The black analog ROI should be 7.5, not 5,8. Is there anything I can do to improve the readings?

![Screenshot from 2024-03-28 00-34-03](https://github.com/jomjol/AI-on-the-edge-device/assets/116210/476b5e89-d0bd-49f2-bbbb-21095823a5d7)

**Comment by SybexX** [comment]
I think it's your LED ring, you can see it fully in the picture(the reflection of it).

**Comment by jalla2000** [comment]
Dang. It's so hard to not get reflections....

**Comment by SybexX** [comment]
Maybe a thin white paper in front of the LED ring or a piece of Frosted Film could help.
or take a look here: https://github.com/jomjol/AI-on-the-edge-device/discussions/1949
